mail_connector_queue/job.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp.addons.connector.event import on_record_create
+from openerp.addons.connector.queue.job import job
+
+
+@job
+def send_mail(session, mail_id):
+    mail = session.env['mail.mail'].browse(mail_id)
+    if mail.state == 'outgoing':
+        mail.send(auto_commit=True, raise_exception=True)
+    else:
+        return "Not in Outgoing state, ignoring"
```

Comment:
is this returned text displayed somewhere?

Reply:
Yes, it's shown in the Job form.

Reply:
@simahawk I considered it, but using which language? I'm not sure it makes sense to use the language of the user sending the email, since it's essentially an internal message. Arguably it should be translated to the language of the user seeing the job form, but that strikes me as adding unnecessary complexity.

Reply:
hmm, indeed this is going to be stored and all in all is just for advanced users: ignore me :stuck_out_tongue_winking_eye: